README.md

```diff
@@ -1,13 +1,18 @@
 <a name="readme-top"></a>
 
+<div align="center">
+  <img src="https://github.com/microverseinc/readme-template/blob/master/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+</div>
+
```

Comment:
# Optional
- I strongly advise you to remove the logo(Microverse). You can replace it with your own logo if you have it. This is part of the general requirement.